pint/utils.py

```diff
@@ -288,6 +289,33 @@ def str2longdouble(str):
     return str2ldarr1(str)[0]
 
 
+def split_prefixed_name(name):
```

Comment:
Isn't this just a one-liner doing something like the following with a better regular expression?
int(re.match(r'([a-z]+_*)(\d+)', name, re.I).groups()[-1])
If you wanted the prefix string and the value, you could do:

def split_prefixed_name(name):
    s, v = re.match(r'([a-z]+_*)(\d+)', name, re.I).groups()
    return s, int(v)